xonsh/prompt_toolkit_shell.py

```diff
@@ -110,21 +111,66 @@ def _get_prompt_tokens_and_style(self):
         def get_tokens(cli):
             return list(zip(tokens, strings))
 
-        class CustomStyle(Style):
-            styles = DefaultStyle.styles.copy()
-            styles.update({
-                Token.Menu.Completions.Completion.Current: 'bg:#00aaaa #000000',
-                Token.Menu.Completions.Completion: 'bg:#008888 #ffffff',
-                Token.Menu.Completions.ProgressButton: 'bg:#003333',
-                Token.Menu.Completions.ProgressBar: 'bg:#00aaaa',
-                Token.AutoSuggestion: '#666666',
-                Token.Aborted: '#888888',
-            })
-            # update with the prompt styles
-            styles.update({t: s for (t, s) in zip(tokens, cstyles)})
-            # Update with with any user styles
-            userstyle = builtins.__xonsh_env__.get('PROMPT_TOOLKIT_STYLES')
-            if userstyle is not None:
-                styles.update(userstyle)
-
-        return get_tokens, CustomStyle
+        custom_style = _xonsh_style(tokens, cstyles)
+
+        return get_tokens, custom_style
+
+
+def _xonsh_style(tokens=[], cstyles=[]):
```

Comment:
Keyword argument values should not be mutable, unless this is on purpose (which it doesn't seem to be).  Could you replace with tuples, please.